src/main/java/uk/gov/hmcts/reform/wataskmanagementapi/auth/role/entities/request/RoleAssignmentRequest.java

```diff
@@ -0,0 +1,58 @@
+package uk.gov.hmcts.reform.wataskmanagementapi.auth.role.entities.request;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.google.common.base.Objects;
+import uk.gov.hmcts.reform.wataskmanagementapi.auth.role.entities.Assignment;
+
+import java.util.List;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
from what I can tell, I think this is the default Jackson object mapper configuration in spring-boot. 
See here: https://docs.spring.io/spring-boot/docs/current/reference/html/howto.html#howto-customize-the-jackson-objectmapper